spec/acceptance/basic_dsc_resources/service/service_spec.rb

```diff
@@ -0,0 +1,128 @@
+require 'spec_helper_acceptance'
+
+describe 'Service tests' do
+
+  context 'MODULES-2336 - C68749 - Attempt to Apply DSC Service Resource with Incompatible "StartupType" and "State"' do
```

Comment:
The "C68749" is a TestRail id, which we're not going to be using in the future so we can get rid of that. Also the JIRA ticket is kind of distracting in output, so maybe move that to a comment.